src/main/java/gregtech/api/unification/material/properties/FissionFuelProperty.java

```diff
@@ -0,0 +1,81 @@
+package gregtech.api.unification.material.properties;
+
+public class FissionFuelProperty implements IMaterialProperty {
+
+    private int maxTemperature;
+    private int duration;
+    private double slowNeutronCaptureCrossSection;
```

Comment:
some documentation(either in the docs or in code) as to what these values affect?

Reply:
Done in the code. It is, again, a little in the weeds, and also I did have to deviate from the real values for the actual fuel rods, so it would make very little sense.